test/common/secret/secret_manager_impl_test.cc

```diff
@@ -50,6 +50,82 @@ name: "abc.com"
             secret_manager->findStaticTlsCertificateProvider("abc.com")->secret()->privateKey());
 }
 
+TEST_F(SecretManagerImplTest, DuplicateStaticTlsCertificateSecret) {
+  envoy::api::v2::auth::Secret secret_config;
+
+  std::string yaml =
```

Comment:
Nit: `const std::string`

Reply:
Add const into this file and context_impl_test.cc. Thanks.